modules/authentication/src/handlers/local.ts

```diff
@@ -756,6 +773,45 @@ export class LocalHandlers implements IAuthenticationStrategy {
     return 'Email changed successfully';
   }
 
+  async resendVerificationEmail(
+    call: ParsedRouterRequest,
+  ): Promise<UnparsedRouterResponse> {
+    if (!this.sendVerificationEmail) return 'Configuration not set to send email';
+
+    let { email } = call.request.params;
+    if (AuthUtils.invalidEmailAddress(email)) {
+      throw new GrpcError(status.INVALID_ARGUMENT, 'Invalid email address provided');
+    }
+    email = email.toLowerCase();
+    const user: User | null = await User.getInstance().findOne({
+      email: email,
+    });
+    if (isNil(user)) throw new GrpcError(status.NOT_FOUND, 'User not found');
+    const verificationToken: Token | null = await Token.getInstance().findOne({
```

Comment:
We should probably recreate these instead, replacing any existing ones of that type for target user.
What if we let these expire down the line?
What if token creation failed for whatever reason, in which case we're stuck with a non-verifiable user.